src/Helpers/Tap/TappedNavigationTab.ts

```diff
@@ -0,0 +1,28 @@
+import { ActionType, TappedNavigationTab } from "../../Schema"
+
+/**
+ * A user taps a navigation tab on iOS
+ *
+ *  @example
+ *  ```
+ *  tappedNavigationTab({
+ *    context_module : "exhibitorsView",
+ *    context_screen_owner_type: "fair",
+ *    context_screen_owner_id: "5f4d80c972737e000deff1ed",
+ *    context_screen_owner_slug: "latitude-art-fair-2020",
+ *    subject: "View more"
```

Comment:
@mikehrom LMK if there's a better example text to use for `subject`.

Reply:
I think this is fine! In the original use case (fairs) we didn't need a subject, but in the Inbox use case, it will be "Bids" or "Inquiries" -- beyond that, it could be a number of options.